docs/_data/prague-2021-config.yaml

```diff
@@ -0,0 +1,172 @@
+    # - text: See the Schedule!
+    #   link: /schedule
+#     - text: Date and ticket information
+#       link: /news/announcing-virtual-conf-dates-tickets
+     - text: Sponsor the conference
+       link: /sponsors/prospectus
+    # - text: See the talks!
+    #   link: /speakers
+#     - text: Buy a Ticket!
+#       link: /tickets
+    # - text: Prepare with the Welcome Wagon
+    #   link: /welcome-wagon
+    # - text: See the schedule!
+    #   link: /schedule
+    # - text: Watch the Live Stream
+    #   link: /livestream
+    # - text: Browse the photos
+    #   link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+    # - text: Watch the videos!
+    #   link_absolute: https://www.youtube.com/playlist?list=PLZAeFn6dfHpkBJAPYFrob6gqdiBuePwGJ
+
+tickets:
+  corporate:
+    price: €125
+  independent:
+    price: €75
+  student:
+    price: €25
+
+sponsorship:
+  second_draft:
+    price: €1,500
+  publisher:
+    price: €3,500
+  patron:
+    price: €7,500
+  keystone:
+    price: €13,500
+
+
+date: # how do we handle these? human readable would be nice.
+  main: "**April 25-27, 2021, online**"
+  short: Oct 3-5, 2021
+  tickets_live: "XXXX 2021"
+  month: October
+  # Conference is special for the index to show the combined conference days
+  conference:
+    event: Main Conference
+    date: October 4-5
+    summary: The main days of the conference. We will be running one main track, and an <a href="/conf/prague/2021/unconference/">Unconference</a>.
+    icon: conference
+  # These are listed in the schedule
+  day_one:
+    event: Hike
+    date: On your own
+    summary: We encourage you to go on a hike in your own beautiful home town.
+    #summary: If you're in town early, join us for the Hike. prague has excellent access to the woods right in the city, and we'll be exploring Forest Park and the Hoyt Arboretum.
+    icon: hike
+    dotw: Saturday
+  day_two:
+    event: Writing Day
+    date: October 3
+    summary: Join us for the <a href="/conf/prague/2021/writing-day/">Writing Day</a> and Welcome Reception. The first official day of the conference is full of chances to interact with other documentarians.
+    #summary: Join us for the Writing Day and Welcome Reception. The first official day of the conference is full of chances to interact with other documentarians.
+    icon: writing
+    dotw: Sunday
+    writing_day_time: 10:00-18:00
+  day_three:
+    event: Conference Day 1
+    date: October 4
+    summary: This is the main event! Hear from lots of interesting folks about all things documentation. We will have talks all day, and unconference sessions running in parallel.
+    dotw: Monday
+    talk_time: 10:00-18:00
+    unconference_time: 9:45-17:00
+  day_four:
+    event: Conference Day 2
+    date: October 5
+    summary: And the conference goes on! We are already starting to miss you. |:heart:|
+    dotw: Tuesday
+    talk_time: 10:00-18:00
+    job_fair_time: 9:45-12:00
```

Comment:
```suggestion
    job_fair_time: 10:00-12:00
```

Reply:
I think since the first talk is a regular talk we can start the job fair at the same time